src/firebase/database.ts

```diff
@@ -0,0 +1,50 @@
+import { getFirestore, Timestamp, getDocs, addDoc, collection, query, where } from "firebase/firestore";
+
+interface resume {
+  description: string | null,
+  url: string,
+  uid: string,
+  jobId: string | null,
+  filename: string
+}
+
+export const addResume = async (data: resume) => {  
+  try {
+    const db = getFirestore();
+    await addDoc(collection(db, "resumes"), {
+      ...data, 
+      "dateCreated": Timestamp.fromDate(new Date())
+    })
+  } catch (error) {
+    console.error(`Something went wrong while trying to add resume for user(${data.uid}): ${error}`)
+    throw new Error(`${error}`)
+  }
+}
+
+export const getUserResume = async (uid: string, filename: string) => {
+  try {
+    const db = getFirestore();
```

Comment:
repetitive and reinitializing this multiple times, use the context in a component and then pass in from the component into this function